examples/sn-blog-gatsby/src/pages/index.tsx

```diff
@@ -58,7 +59,6 @@ export interface IndexPageProps {
 const IndexPage: React.FC<IndexPageProps> = (props) => {
   const { data } = props
   const classes = useStyles()
```

Comment:
- padding on blogItem is removable, it is overridden anyway
- link: `display: flex; flex-direction;` why?